File: src/main/java/carelog/carelog/relation/web/dto/RelationStatusUpdateRequest.java
package carelog.carelog.relation.web.dto;

import carelog.carelog.relation.domain.*;
import jakarta.validation.constraints.*;
import lombok.*;

@Getter
@NoArgsConstructor
public class RelationStatusUpdateRequest {

    @NotNull(message = "상태는 필수 입력 값입니다.")
    private RelationStatus status;
}

File: src/main/kotlin/carelog/carelog/relation/web/dto/RelationStatusUpdateRequest.kt
package carelog.carelog.relation.web.dto

import carelog.carelog.relation.domain.RelationStatus
import io.swagger.v3.oas.annotations.media.Schema
import jakarta.validation.constraints.NotNull

data class RelationStatusUpdateRequest(
    @field:NotNull(message = "상태는 필수 입력 값입니다.")
    @Schema(description = "변경할 관계 상태")
    val status: RelationStatus,
)
